packages/mobile/src/apollo/__mockData__.ts

```diff
@@ -1,93 +1,81 @@
+import { TransactionTypes, TransferTransactionTypes } from 'src/transactions/reducer'
 
 export const invitedAddress = '0x1b173'
 
 const exchangeDollar: HomeExchangeFragment = {
-  __typename: EventTypeNames.Exchange,
-  type: 'EXCHANGE' as TransactionTypes,
+  type: 'EXCHANGE' as TransactionTypes.EXCHANGE,
   hash: '1',
   inValue: 19080,
   timestamp: Date.now(),
-  inSymbol: 'Celo Dollar',
-  outSymbol: 'Celo Gold',
+  inSymbol: 'Celo Dollar' as CURRENCY_ENUM,
+  outSymbol: 'Celo Gold' as CURRENCY_ENUM,
   outValue: 62252,
 }
 
 const exchangeGold: HomeExchangeFragment = {
-  __typename: EventTypeNames.Exchange,
-  type: 'EXCHANGE' as TransactionTypes,
+  type: 'EXCHANGE' as TransactionTypes.EXCHANGE,
   hash: '1',
   inValue: 190,
   timestamp: Date.now(),
-  inSymbol: 'Celo Gold',
-  outSymbol: 'Celo Dollar',
+  inSymbol: 'Celo Gold' as CURRENCY_ENUM,
+  outSymbol: 'Celo Dollar' as CURRENCY_ENUM,
   outValue: 62,
 }
 
 const sent: HomeTransferFragment = {
-  __typename: EventTypeNames.Transfer,
-  type: 'SENT' as TransactionTypes,
+  type: 'SENT' as TransferTransactionTypes,
   value: 987161,
-  symbol: 'Celo Gold',
+  symbol: 'Celo Gold' as CURRENCY_ENUM,
   timestamp: Date.now(),
   address: '0x423043cca38e75d7913504fedfd1dd4539cc55b3',
   comment: 'FAKE FAKE FAKE',
   hash: '01010',
 }
 
 const sentInvite: HomeTransferFragment = {
-  __typename: EventTypeNames.Transfer,
-  type: 'SENT' as TransactionTypes,
+  type: 'SENT' as TransferTransactionTypes,
   value: 0.33,
-  symbol: 'Celo Dollar',
+  symbol: 'Celo Dollar' as CURRENCY_ENUM,
   timestamp: Date.now(),
   address: invitedAddress,
   comment: SENTINEL_INVITE_COMMENT,
   hash: '0x1010',
 }
 
 const recieved: HomeTransferFragment = {
-  __typename: EventTypeNames.Transfer,
-  type: 'RECEIVED' as TransactionTypes,
+  type: 'RECEIVED' as TransferTransactionTypes,
   value: 587161,
-  symbol: 'Celo Gold',
+  symbol: 'Celo Gold' as CURRENCY_ENUM,
   timestamp: Date.now(),
   address: '0x423043cca38e75d7913504fedfd1dd4539cc55b3',
   comment: 'FAKE FAKE FAKE',
   hash: '01010',
 }
 const faucet: HomeTransferFragment = {
-  __typename: EventTypeNames.Transfer,
-  type: 'FAUCET' as TransactionTypes,
+  type: 'FAUCET' as TransferTransactionTypes,
   value: 387161,
-  symbol: 'Celo Dollar',
+  symbol: 'Celo Dollar' as CURRENCY_ENUM,
   timestamp: Date.now(),
   address: '0x423043cca38e75d7913504fedfd1dd4539cc55b3',
   comment: 'FAKE FAKE FAKE',
   hash: '01010',
 }
 const verificationFee: HomeTransferFragment = {
-  __typename: EventTypeNames.Transfer,
-  type: 'VERIFICATION_FEE' as TransactionTypes,
+  type: 'VERIFICATION_FEE' as TransferTransactionTypes,
   value: 0.3,
-  symbol: 'Celo Gold',
+  symbol: 'Celo Gold' as CURRENCY_ENUM,
   timestamp: Date.now(),
   address: '0x423043cca38e75d7913504fedfd1dd4539cc55b3',
   comment: 'FAKE FAKE FAKE',
   hash: '01010',
 }
 const verificationReward: HomeTransferFragment = {
-  __typename: EventTypeNames.Transfer,
```

Comment:
Why don't we need these graphQL provided __typename properties anymore? Is it cause @jeanregisser split the home feed and exchange feed queries in bc-api?

Reply:
No, since we know the EventType from the transaction type